compliance_checker/tests/test_cf.py

```diff
@@ -2449,6 +2452,59 @@ def test_taxonomy_skip_lsid(self):
         assert len(results[0].msgs) == 0
         assert results[0].value[0] == results[0].value[1]
 
+        
+class TestCF1_9(BaseTestCase):
+    def setUp(self):
+        self.cf = CF1_9Check()
+
+    def test_check_data_types(self):
+        dataset = MockTimeSeries()
```

Comment:
Ideally other data types should also be checked.  I wanted to keep DRY, but couldn't figure out how to inherit the tests from CF 1.6 data type checks without inheriting from the test case for 1.6.  Any suggestions would be welcome.